webpack/assets/javascripts/foreman_hosts.js

```diff
@@ -0,0 +1,39 @@
+export let pluginEditAttributes = {
```

Comment:
@matanwerbner  if you are interested to know how this is used in katello
https://github.com/Katello/katello/pull/6746/files